HHS/Head-Start-TTADP
# [TTAHUB-760] Always Maintain Region Filters in URL

## Description of change

We always want to maintain the region filters in the URL. This will ensure that when a URL filter is shared the user will be correctly notified if they are attempting to view a region they can't access.

## How to test

Test on both AR Landing and Regional Dashboard pages. 

Test for both a regular and Central Office user.

1. View the landing/dashboard page in a new browser. The URL should show all user regions.
2. Add a custom region filter. The URL should only show the region specified.
3. Remove the region filter. The URL should add back all regions the user can access.
4. Create a URL that has SOME of the regions the user can access. Paste it in a new browser window. You should see pills for the regions in the URL filter. 
5. Create a URL filter that has ALL of the regions the user can access. Paste it in a new browser window. There shouldn't be any filter pills for region but the URL should reflect all regions.

All preexisting filter functionality should still work as expected.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-684


## Checklists

### Every PR


~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy


### After merge/deploy
